### PR TITLE
sci-physics/herwig: version bump (based on herwig++ branch)

### PR DESCRIPTION
**Pull request by jauhien**
I ask for review because it's quite a big change: herwig is now based on the code previously known as herwig++. It was renamed to herwig with the latest release and old fortran branch of herwig is now depricated. Please, review this ebuild. If no comments I will proceed with merging it in a week.

### CODE REVIEW

**Comment by jauhien** [comment]
@bircoph or somebody else interested in Physics, please, have a look.

**Comment by monsieurp** [comment]
@jlec maybe?

**Comment by monsieurp** [comment]
@jauhien have you got round to taking a look at @bircoph's pointers?

**Comment by jauhien** [comment]
@monsieurp: yes, I've looked at them and will push ebuild changed
accordingly. I had not enough time to proceed with this until now though.

2016-02-03 9:36 GMT+01:00 monsieurp <notifications@github.com>:

**Comment by monsieurp** [comment]
@jauhien ping

File: sci-physics/herwig/herwig-7.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```
Comment (jlec) [inline sci-physics/herwig/herwig-7.0.0.ebuild]: Please use EAPI=6

**Comment by monsieurp** [comment]
ping?

**Comment by jauhien** [comment]
Hi,

sorry, I'm quite busy with real life at the moment. I hope to return to
more active development this month and this version bump is the first
thing that I'm going to finish.

**Comment by monsieurp** [comment]
@jauhien feel free to reopen this PR when you'll get round to working on it.

**Pull request closed without merging**